go.mod

```diff
@@ -172,8 +172,6 @@ require (
 replace k8s.io/client-go => k8s.io/client-go v0.26.4
 
 replace (
-	github.com/metal3-io/baremetal-operator/apis => github.com/openshift/baremetal-operator/apis v0.0.0-20211201170610-92ffa60c683d
-	github.com/metal3-io/baremetal-operator/pkg/hardwareutils => github.com/openshift/baremetal-operator/pkg/hardwareutils v0.0.0-20211201170610-92ffa60c683d
```

Comment:
Was it removed by `go mod tidy`? How these lines were added here?

Reply:
If you note, these lines are in the replace section of go.mod and will not be automatically removed by `go mod tidy`. The go.mod and go.sum files don't have any other reference to baremetal-operator which tells me that we probably removed the dependency on that long back.

Reply:
Right, makes sense.